.env

```diff
@@ -32,28 +32,28 @@ DATABASE_URL=postgres://postgres:postgrespass@localhost:5432/postgres?schema=api
 ########
 S3_ENDPOINT=https://cdn-dev.podkrepi.bg
 S3_BUCKET=campaign-files
-S3_ACCESS_KEY=
-S3_SECRET_ACCESS_KEY=
+S3_ACCESS_KEY=s3-access-key
+S3_SECRET_ACCESS_KEY=s3-secret-access-key
 
 ## Keycloak ##
 ##############
 KEYCLOAK_URL=https://keycloak.podkrepi.bg/auth
 KEYCLOAK_REALM=webapp-dev
 KEYCLOAK_CLIENT_ID=jwt-headless
-KEYCLOAK_SECRET=
+KEYCLOAK_SECRET=99043b92-c337-4869-8a3c-09822db0fd6c
```

Comment:
@GeorgyGenchev Exposing a secret key is not acceptable. Please undo this change and be careful next time you commit something.

In order to pass secrets to the project the first step is to copy `.env.local.example` to `.env.local` and add you secrets there.

Read more at https://nextjs.org/docs/basic-features/environment-variables#environment-variable-load-order